app/dashboard/templates/funder_dashboard.html

```diff
@@ -0,0 +1,808 @@
+                            class="funder-dashhboard__payout-history__chart"
+                            style="width: 100%; height: 15rem;">
+                    </canvas>
+                  </div>
+                </div>
+                <div class="funder-dashboard__tax-reporting row card mt-1">
+                  <div class="card-body pb-2">
+                    <div class="funder-dashboard__tax-reporting__title">
+                      <h4 class="text-capitalize pb-1 pt-0 text-fd-no-spacing d-inline-block mr-1">
+                        {% trans "Tax Reporting" %}
+                      </h4>
+                      <span class="text-secondary">
+                        {% trans "Amount to be reported for "%}
+                        <span class="funder-dashboard__tax-reporting__year">2018</span>
+                      </span>
+                    </div>
+                    <div class="funder-dashboard__tax-reporting__bounties
+                                d-flex align-items-center justify-content-between">
+                      <p class="lead pt-0">
+                        <span class="funder-dashboard__tax-reporting__bounties__count">
+                          139
+                        </span>
+                        <span class="funder-dashboard__tax-reporting__bounties__print pr-1">
+                          {% trans "Bounties" %}
+                        </span>
+                        <span class="funder-dashboard__tax-reporting__bounties__worth pl-1">
+                          $34,450
+                        </span>
+                      </p>
+                      <div class="mb-2">
+                        <button class="button text text-white" style="background-color: #8e2abe;">
+                          <i class="fas fa-download pr-1">
+                          </i>
+                          {% trans "Download Report" %}
+                        </button>
+                      </div>
+                    </div>
+                    <div class="d-flex justify-content-end">
+                      <a href="#">
+                        {% trans "View All Bounties" %}
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="funder-dashboard__latest-on-bounties
+                          col card p-1 pt-0 ml-3">
+                <div class="card-body">
+                  <div class="text-capitalize lead">
+                    {% trans "Latest on Your Bounties" %}
+                  </div>
+                  <hr>
+                  <div class="funder-dashboard__latest-on-bounties__reclaim-funds card">
+                    <div class="card-body pb-1">
+                      <div class="text-capitalize lead">
+                        {% trans "Reclaim Your Funds" %}
+                      </div>
+                      <div>
+                        You have
```

Comment:
Can we wrap this and all of the other remaining relevant strings  in `trans` throughout this file ?